tests/api_workflow/test_api_workflow_datasources.py

```diff
@@ -50,6 +50,23 @@ def test_download_raw_samples_equal_to_download_all_raw_new_samples(self):
         assert len(samples) == len(new_samples)
         assert set(samples) == set(new_samples)
 
+    def test_download_raw_samples_or_metadata_relevant_filenames(self):
+        self.api_workflow_client._datasources_api.reset()
+        for method in [
+            self.api_workflow_client.download_raw_samples,
+            self.api_workflow_client.download_raw_metadata
+        ]:
+            for relevant_filenames_path in [None, "", "relevant_filenames.txt"]:
+                with self.subTest(
+                        relevant_filenames_path=relevant_filenames_path,
+                        method=method
+                ):
+                    samples = self.api_workflow_client.download_raw_samples(
```

Comment:
shouldn't you be using the method you are passing in? 🤔

Reply:
Good point, this is an error.